rmgpy/molecule/molecule.py

```diff
@@ -36,28 +36,34 @@
 describe the corresponding atom or bond.
 """
 
-import cython
+from __future__ import division
+
+import itertools
 import logging
 import os
-import numpy
-import urllib
 from collections import OrderedDict, defaultdict
-import itertools
 from copy import deepcopy
+try:
+    from urllib.parse import quote
```

Comment:
Yes, it's because the function was relocated. I added comments clarifying which syntax is for which Python version. The plan is to remove the Python 2 syntax when we're done with the transition.